src/plcc/parse_spec/parse_syntactic_spec/parse_syntactic_spec.py

```diff
@@ -0,0 +1,133 @@
+from typing import List
+from plcc.load_spec.load_rough_spec.parse_dividers import Divider
+from plcc.load_spec.load_rough_spec.parse_lines import Line
+import re
+from re import Match
+from .parse_syntactic_structs import (
+    SyntacticSpec,
+    SyntacticRule,
+    Symbol,
+    CapturingTerminal,
+    RepeatingSyntacticRule,
+    StandardSyntacticRule,
+    LhsNonTerminal,
+    RhsNonTerminal,
+    Terminal,
+    CapturingSymbol,
+    MalformedLHSError,
+    MalformedBNFError,
+)
+
+
+def parse_left(line: Line, lhs: str) -> LhsNonTerminal:
+    match = re.match(r"<(?P<non_terminal>\S*)>(?::(?P<alt_name>\S+))?\s*", lhs)
+    if match is None:
+        raise MalformedLHSError(line)
+
+    non_terminal = match["non_terminal"]
+    alt_name = match["alt_name"]
+    return LhsNonTerminal(non_terminal, alt_name)
+
+
+def parse_right(line: Line, rhs: str) -> List[Symbol]:
+    total = []
+    if not rhs:
+        return total
+    symbols = rhs.split(" ")
+    for symbol in symbols:
+        if symbol == "":
+            continue
+        if symbol.startswith("#") or symbol is None:
+            return total
+        total.append(parse_symbol(symbol))
+    return total
+
+
+def parse_symbol(symbol: str) -> Symbol:
+    capturing = re.match(r"<(?P<name>\S*)>(?P<alt_name>\S+)?", symbol)
+
+    if capturing:
+        name = capturing["name"]
+        alt_name = capturing["alt_name"]
+        return parse_capturing(name, alt_name)
+    return Terminal(symbol)
```

Comment:
This can be compressed to shorten the function (Note that I am using a class-based approach in my refactor, along with camelCase): 
```
return self.parseCapturing(capturing["name"], capturing["alt_name"]) if capturing else Terminal(symbol)
```